src/js/components/validateData/validateValues/ValidateValuesFileComponent.jsx

```diff
@@ -84,6 +85,23 @@ export default class ValidateDataFileComponent extends React.Component {
         return stagedFile;
     }
 
+    downloadFile(item){
+        console.log(item);
+        if(item.file_type=='appropriations'){
+            GenerateFilesHelper.fetchFile('A', this.props.submission.id).then((result)=>{
+                window.location = result.url;
+            })
+        }else if(item.file_type=='program_activity'){
+            GenerateFilesHelper.fetchFile('B', this.props.submission.id).then((result)=>{
+                window.location = result.url;
+            })
+        }else if(item.file_type=='award_financial'){
+            GenerateFilesHelper.fetchFile('C', this.props.submission.id).then((result)=>{
+                window.location = result.url;
+            })
+        }
+    }
```

Comment:
You might want to include a catch for errors. Also, some sign that it's working (like in the sample I pointed you to with the `signInProgress` example) might be nice. If it's taking a while people might click multiple times. We only want to sign it if we aren't already in the process of doing so. Specifically look at `clickedReport` and `signReport` for what I'm talking about.